AutoRunCustomTool64/AutoRunCustomTool64.csproj

```diff
@@ -62,6 +58,16 @@
     <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.0.31902.203" ExcludeAssets="runtime" />
     <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.0.5234" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="..\Shared\Resources\Package.ico">
+      <Link>Package.ico</Link>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+      <IncludeInVSIX>true</IncludeInVSIX>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
```

Comment:
This folder is now empty, so it's not necessary to keep it

Reply:
Strange that Visual Studio didn't do that for me... oh well, it's gone now.